src/service/alert_manager.rs

```diff
@@ -47,11 +45,14 @@ pub async fn handle_triggers(alert_name: &str, trigger: Trigger) {
     )
     .await
     {
+        Err(_) => log::error!("[ALERT MANAGER] Error fetching alert"),
```

Comment:
`match` and `if` expressions are easier to read when they start with short branches — those can be dealt with quickly, allowing the reader to focus on more complicated branches.